velox/exec/Operator.cpp

```diff
@@ -295,77 +295,86 @@ void Operator::recordBlockingTime(uint64_t start, BlockingReason reason) {
       fmt::format("blocked{}Times", blockReason), RuntimeCounter(1));
 }
 
-void Operator::recordSpillStats(const common::SpillStats& spillStats) {
+void Operator::recordSpillStats() {
```

Comment:
Let's call this once on operator close? Otherwise you need to call reset at the end of this call to clear spillStats_. I felt it is ok to just call this once on close for now to get the runtime stats at the end of operator or query execution.